jobs/JGLOBAL_ATMOS_POST

```diff
@@ -68,30 +73,32 @@ export COMPONENT=${COMPONENT:-atmos}
 ##############################################
 # Begin JOB SPECIFIC work
 ##############################################
-export APRUNP=${APRUN:-$APRUN_NP}
+export APRUNP=${APRUN:-${APRUN_NP}}
 export RERUN=${RERUN:-NO}
 export HOMECRTM=${HOMECRTM:-${PACKAGEROOT:?}/lib/crtm/${crtm_ver:?}}
 export FIXCRTM=${CRTM_FIX:-${HOMECRTM}/fix}
-export PARMpost=${PARMpost:-$HOMEgfs/parm/post}
+export PARMpost=${PARMpost:-${HOMEgfs}/parm/post}
 export INLINE_POST=${WRITE_DOPOST:-".false."}
 
-export COMIN=${COMIN:-$ROTDIR/$RUN.$PDY/$cyc/$COMPONENT}
-export COMOUT=${COMOUT:-$ROTDIR/$RUN.$PDY/$cyc/$COMPONENT}
+export COMIN=${COMIN:-${ROTDIR}/${RUN}.${PDY}/${cyc}/${COMPONENT}}
```

Comment:
Is there ever a reason for `COMPONENT` in this script to not be `atmos`?
Line 70 should be:
`export COMPONENT="atmos"`

Reply:
Not that I know of. I was just updating for the linter.